novawallet/Modules/Transfer/TransferSetup/TransferSetupPresenter.swift

```diff
@@ -35,6 +36,8 @@ final class TransferSetupPresenter {
         }
     }
 
+    private var isManualFeeSet: Bool = false
```

Comment:
What is this guy for?

Reply:
Forgot to remove it since it became redundant at the end. Thanks!